Autor.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class Autor {
+    public string $nombre;
+    public string $apellido;
+    public int $edad;
+
+    public function __construct(string $nombre, string $apellido, int $edad) {
+        $this->nombre = $nombre;
+        $this->apellido = $apellido;
+        $this->edad = $edad;
+    }
+
+    public function getNombreCompleto(): string {
+        return $this->nombre . " " . $this->apellido;
+    }
+
+    public function getNombre(): string { return $this->nombre; }
```

Comment:
```suggestion
    public function getNombre(): string {
        return $this->nombre;
    }
```
Con este formato es más legible, puede basarse en el estilo de código [PSR-12](https://www.php-fig.org/psr/psr-12/)  sección 4.4 Methods and Functions